src/arch/xtensa/up/xtos/reset-vector.S

```diff
@@ -581,7 +581,7 @@ unpackdone:
 
 #if CONFIG_BOOT_LOADER
 		/*ToDo refine the _start*/
-		movi a0, SOF_TEXT_START
+		movi a0, SOF_TEXT_BASE
```

Comment:
#ifdef SUECREEK ? Why is this needed.

Reply:
This is needed because that's where the linker script puts the text section:
```
  sof_text :
        org = SOF_TEXT_BASE,
        len = SOF_TEXT_SIZE,
```
```
#define L2_VECTOR_SIZE		0x2000

#define SOF_TEXT_START		(HP_SRAM_BASE + L2_VECTOR_SIZE)
#define SOF_TEXT_START_SIZE	0x400
#define SOF_TEXT_BASE		(SOF_TEXT_START + SOF_TEXT_START_SIZE)
```
Output of `objdump -h`:
` 10 .text         00012f62  be002400  be002400  00001400  2**4`
The other 3 platforms, that use BOOTLOADER:
`AM_CONDITIONAL(BUILD_BOOTLOADER,  test "$FW_NAME" = "apl" -o "$FW_NAME" = "cnl" -o "$FW_NAME" = "icl" -o "$FW_NAME" = "sue")`
have:
```
#define SOF_TEXT_BASE		(SOF_TEXT_START)
```
So, so far no `#ifdef ` seems to be needed?